vagrant/provisioning/roles/alfresco-site/tasks/main.yml

```diff
@@ -70,14 +70,18 @@
     password: "{{ alfresco_admin_password }}"
   register: rm_container_out
 
+- name: install jmespath for python3.9
+  become: yes
+  command: python3.9 -m pip install jmespath
+
```

Comment:
Do we need for all OS's or just Rocky?

Reply:
jmespath we need on all systems, the Alfresco site and folder role uses it.  I have to install it manually on new sites, before I can finish the alfresco role.